linkatos/message.py

```diff
@@ -1,3 +1,5 @@
+#! /bin/env python
```

Comment:
This is wrong. A shebang is intented for executable files. Not for modules nor scripts executed via `python foo.py`.